src/AutoRest.CSharp/build/CodeGeneration.targets

```diff
@@ -17,14 +17,15 @@
     <AutoRestSharedCodeDirectory>$(MSBuildThisFileDirectory)../content/Generator.Shared/</AutoRestSharedCodeDirectory>
     <ManagementSharedCodeDirectory>$(MSBuildThisFileDirectory)../content/Management.Shared/</ManagementSharedCodeDirectory>
 
-    <_GenerateCode Condition="'$(AutoRestInput)' != ''">true</_GenerateCode>
+    <_GenerateCode Condition="'$(AutoRestInput)' != '' OR '$(CadlInput)' != ''">true</_GenerateCode>
```

Comment:
just out of curiosity, where do we determine the variable `CadlInput` here?

Reply:
This is to set the input cadl file.